### PR TITLE
Allow to edit bug comments

### LINKED ISSUE #4: Interface to edit Issues
The WebUI must allow a user to edit an Issue.
Access right are currently ignored like in the TermUI.

Edit: Editing the issue title is already implemeneted. Lets also allow to edit comments.
E.g. add a pencil icon to comment-header. If a user clicks on this icon, the message should be editable.

### PR DESCRIPTION
**Pull request by GlancingMind**
Closes #4

### CODE REVIEW

File: api/graphql/models/gen_models.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"strconv"
 
 	"github.com/MichaelMure/git-bug/bug"
+	"github.com/MichaelMure/git-bug/entity"
```
Comment (GlancingMind) [inline api/graphql/models/gen_models.go:11]: This file is auto generated and shouldn't be edited.
Go into _api/graphql_ directory and run `go generate` to update this file

File: api/graphql/schema/mutations.graphql [hunk 2]
```diff
@@ -42,6 +42,28 @@ type AddCommentPayload {
...
 }
 
+input EditCommentInput {
+    """A unique identifier for the client performing the mutation."""
+    clientMutationId: String
+    """"The name of the repository. If not set, the default repository is used."""
+    repoRef: String
+    """The bug ID's prefix."""
+    prefix: String!
+    """The new message to be set."""
```
Comment (GlancingMind) [inline api/graphql/schema/mutations.graphql:52]: This here is missing the `target` field for `EditCommentRaw` in `api/graphql/resolvers/mutation.go`

Comment (GlancingMind) [reply api/graphql/schema/mutations.graphql:52]: Setting target type to String will result in following error:
`cannot use input.Target (type string) as type entity.Id in argument to b.EditCommentRaw`
But using entity.Id as Type won't work either as entity.Id is unkown on generation and importing via the go import directive won't work obviously.
How to resolve this? 
@MichaelMure do you have an idea? Maybe I'm on a complete wrong track :disappointed:

File: api/graphql/models/gen_models.go (hunk 1 above)
Comment (GlancingMind) [reply api/graphql/models/gen_models.go]: Resolved.

File: api/graphql/schema/mutations.graphql (hunk 2 above)
Comment (MichaelMure) [reply api/graphql/schema/mutations.graphql:54]: huhu, that happens.

Piggy-backing on that quite useless comment as I don't have another way to contact you, did you receive my email?

Comment (GlancingMind) [reply api/graphql/schema/mutations.graphql:54]: Oh sorry, I look rather rarely into my mail. Will take a look.

**Pull request merged**